### ISSUE TITLE
Перенести миссию на Алтисе в 3D редактор

### ISSUE THREAD

**Issue by malashin** [ToxaBes/ADR-Spec-Ops#112]

**Comment by 12st** [comment]
Если я правильно помню, тоха перед отъездом перевёл в 3D редактор.

**Comment by malashin** [comment]
Нет, не переводил.

https://github.com/ToxaBes/ADR-Spec-Ops/blob/master/co30_ADR_Spec_Ops.Altis/mission.sqm#L1
https://github.com/ToxaBes/ADR-Spec-Ops/blob/master/co30_ADR_Spec_Ops.Tanoa/mission.sqm#L1

**Comment by ToxaBes** [comment]
Я не успел перевести. При попытке перевода он затер все в mission.sqm

**Comment by malashin** [comment]
Да, там нужно портировать всё это дело или искать в чем проблема и править это в 2D версии.

**Comment by ToxaBes** [comment]
Я если честно пока не знаю как там это все портировать. Буду разбираться.

**Comment by Noart537** [comment]
В чем проблемма импортирования в 3D?

**Comment by malashin** [comment]
Ничего не импортируется, просто пустой файл миссии создается.

**Comment by Noart537** [comment]
В 2D зайди, сделай у одного бота (игрового) Атрибут "Игрок" А не "Игровой" сохрани и открывай в 3D

**Comment by ToxaBes** [comment]
Делал так перед отъездом. Не прокатило почему-то.

**Comment by Noart537** [comment]
Должно получиться, попробуйте. Это давнишний косяк. Арме нужен один главный игрок. Хз зачем.

**Comment by Noart537** [comment]
Там еще шифрование mission.sqm по умолчанию включено, гляньте на всякий.

**Comment by ToxaBes** [comment]
Я сегодня буду разбираться, спасибо за подсказку.

**Comment by malashin** [comment]
Импортировал миссию с игроком. Там некоторые вещи поехали, нужно просмотреть все детально и поправить.

В каталоге оставлен бэкап 2D миссии.

**Comment by ToxaBes** [comment]
Ок, я тогда на тебя задачу переназначу. Вариант для Таноа уже в 3D?

**Comment by Horizon1959** [comment]
Зовите тестить вечером, поковыряем.

**Comment by malashin** [comment]
Таноа изначально в 3D редакторе делал.

**Comment by ToxaBes** [comment]
Т.к. это Алтис, то я займусь задачей.
